package.json

```diff
@@ -65,81 +94,131 @@
     "eslint": "^6.1.0",
+    "eslint-plugin-import": "^2.22.1",
+    "express": "^4.17.1",
+    "express-rate-limit": "^5.1.3",
     "glob": "^7.1.1",
+    "html-webpack-plugin": "^4.3.0",
+    "husky": "^4.3.0",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^24.8.0",
-    "lerna": "^3.16.4",
+    "jsdoc": "^3.6.4",
+    "karma": "^5.1.0",
+    "karma-chrome-launcher": "^3.1.0",
+    "karma-coverage": "^2.0.2",
+    "karma-jasmine": "^3.3.1",
+    "karma-parallel": "^0.3.1",
+    "karma-sourcemap-loader": "^0.3.7",
+    "karma-webpack": "^4.0.2",
+    "lerna": "^3.22.1",
+    "less": "^3.12.2",
+    "less-loader": "^7.0.2",
     "link-parent-bin": "^1.0.0",
+    "loader-utils": "^1.4.0",
+    "mini-css-extract-plugin": "^0.9.0",
+    "one-cerner-style-icons": "github:cerner/one-cerner-style-icons",
+    "postcss-loader": "^3.0.0",
+    "prettier": "^2.0.5",
+    "prop-types": "^15.5.8",
+    "puppeteer": "^5.3.1",
     "raf": "^3.4.1",
     "react": "^16.8.5",
     "react-dom": "^16.8.5",
     "react-test-renderer": "^16.4.2",
     "regenerator-runtime": "^0.13.2",
+    "rimraf": "^3.0.2",
+    "semantic-release": "^17.1.1",
+    "sinon": "^9.2.0",
+    "style-loader": "^2.0.0",
     "stylelint": "^11.0.0",
     "stylelint-config-terra": "^3.2.0",
     "terra-aggregate-translations": "^1.0.0",
     "terra-dev-site": "^6.5.0",
     "terra-disclosure-manager": "^4.9.0",
     "terra-enzyme-intl": "^3.0.0",
+    "terra-theme-context": "^1.0.0",
     "terra-toolkit": "^6.2.0",
+    "url-loader": "^4.1.1",
     "webpack": "^4.30.0",
-    "webpack-cli": "^3.3.2",
-    "webpack-dev-server": "^3.3.1"
+    "webpack-cli": "^3.3.12",
+    "webpack-dev-server": "^3.3.1",
+    "webpack-merge": "^4.2.2"
+  },
+  "scripts": {
+    "build": "webpack --config ./packages/carbon-graphs/webpack/webpack.config.js --env.TYPE=LIB",
+    "build:dist": "webpack --config ./packages/carbon-graphs/webpack/webpack.config.js --env.TYPE=DIST",
+    "bootstrap:hoist": "lerna bootstrap --hoist",
+    "clean": "lerna clean --yes && rm -rf node_modules || true",
+    "clean:install": "npm run clean && npm install",
+    "clean:obsolete-snapshots": "npm test -- -u",
+    "compile": "npm run compile:clean && npm run compile:build",
+    "compile:build": "lerna ls | sed 's/@cerner\\///g' | cut -d ' ' -f1 | xargs -I {} babel packages/{}/src --out-dir packages/{}/lib --copy-files",
+    "compile:clean": "rm -rf packages/**/lib",
+    "compile:src": "lerna run compile",
+    "compile:prod": "rm -rf ./build; NODE_ENV=production webpack -p",
+    "compile:lowlight": "rm -rf ./build; THEME=clinical-lowlight-theme webpack -p",
+    "compile:fusion": "rm -rf ./build; THEME=orion-fusion-theme webpack -p",
+    "danger": "danger ci",
+    "dependency-markdown": "node scripts/dependency-markdown-generator/DependencyMarkdownGenerator.js",
+    "dev": "node packages/carbon-graphs/webpack/dev-server.js",
+    "docs": "node scripts/root-readme-updater/index.js",
```

Comment:
i don't think we need this?

Reply:
Removed: https://github.com/cerner/terra-graphs/pull/5/commits/e2e6439671a83f9ffef94a9c73e210d32084f145